lib/c-api/examples/wasmer-capi-examples-runner/src/lib.rs

```diff
@@ -365,7 +365,7 @@ fn test_run() {
             println!("execute: {command:#?}");
             let output = command
                 .output()
-                .expect(&format!("failed to run {command:#?}"));
+                .unwrap_or_else(|_| panic!("failed to run {command:#?}"));
```

Comment:
Use expect instead

Reply:
this is a clippy autofix done by clippy --fix --all, it does exactly the same thing, but it doesn't construct a string in the "okay" case. wontfix because it will be reapplied by `clippy --fix` on the next run anyways.